CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+### May 6, 2025
+
+- Modified project list to include project filters [DESENG-799](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-799)
```

Comment:
For those items that are related to DESENG-799, could I ask you to indent them under the line with the ticket? That way we know what was needed for the ticket and what was extraneous

Reply:
Sure, I will do that